k8s/prod/cluster-00/em/em-icp.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: em-icp
+  namespace: em
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.nodejs: glob:prod-*
+spec:
+  releaseName: em-icp
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: em-icp
+    version: 0.1.10
+  values:
+    nodejs:
+      replicas: 1
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/em/icp:prod-ae3d2e63
+      ingressHost: em-icp-prod.service.core-compute-prod.internal
+      environment:
+        VAR_T3: trigger4
```

Comment:
This shouldn’t be needed

Reply:
Thank you @timja . I will need to implement this across all our API's. Will implement this review comment as part of separate ticket.